_emails/us/oregon/portland_defund_ppb.md

```diff
@@ -17,34 +17,48 @@ recipients:
+  3.b) Eliminate these positions entirely by reducing the total number of officers, not just by folding them back into general sworn officer positions.
+
+4) Immediately stop policing protests - officers are bringing the riot, not the demonstrators. Even if they stop using tear gas, glass beads, plastic and rubber bullets, and other 'less lethal weapons' they will continue to behave how they know and shift to other ways to enact violence.
+
+5) End PPB relationship to the Federal 1033 program, which supplies militarized weapons, gear, and vehicles to local police departments across the country.
+
+6) Divest City of Portland Cannabis Tax funds from the PPB budget AND invest them instead into reparations for those harmed by the War on Drugs, which disproportionately targeted Black communities for activities that are now legal.
+
+7) Immediately stop expansion of the Portland Police Bureau budget by ending all budget increases. The City Council must commit to never vote for PPB Budget increases again.
+
+8) Immediately end the use of police responders to health or mental health calls and wellness checks, and implement alternative responses that are not led by or contingent on interaction with the police.
+
+9) Immediately terminate contract between the Portland Police Bureau and Portland Business Alliance for the Clean & Safe Enhanced Services District.
+
+10) Immediately eliminate police presence in historically Black neighborhoods, communities of color, and areas with high houseless populations in order to reduce the fear, ongoing trauma, violence, and oppressive conditions which impede people’s daily survival.
+
+Finally, I need to see that there will be allocations to the bereaved families of the citizens of Portland killed by members of the PPB. I urge that the testimonies of those who have come forward be reviewed, as the names of the officers at fault and the victims that have fallen at their hands have been adequately named.
+
+The PPB has been caught in collusion with prominent white nationalist groups, shootings of Black people and people perceived to be in mental health crisis, and this violence and terrorism against our citizens needs to stop NOW.
+
+I call on the Mayor, the Commissioners, and all other City Council Members to use their powers to continue to further reduce the budget and pledge to increase Police accountability and transparency. I promise that you will lose my support and my vote if you aren’t willing to make a public and vocal stand to truly defund the PPB, increase their accountability and transparency, and do due diligence to ensure that we no longer need to use the hashtag Black Lives Matter because it will be evident in our practices.
```

Comment:
```suggestion
I call on the Mayor, the Commissioners, and all other City Council Members to use their powers to continue to further reduce the budget and pledge to increase Police accountability and transparency. I promise that you will lose my support and my vote if you aren’t willing to make a public and vocal stand to truly defund the PPB, increase their accountability and transparency, and do due diligence to ensure that we nobody needs to use the Black Lives Matter hashtag anymore because it will be evident in our practices.
```